xarray/tests/test_dataarray.py

```diff
@@ -2519,6 +2519,69 @@ def test_combine_first(self):
                              [('x', ['a', 'b', 'd']), ('y', [-1, 0])])
         self.assertDataArrayEqual(actual, expected)
 
+    def test_sortby(self):
```

Comment:
Can you delete the test cases here that are redundant with those in `test_dataset.py`? Given the implementation (only a small wrapped for `DataArray.sortby()`), I am OK having a less comprehensive set of unit tests for the DataArray implementation -- and fewer unit tests make things easier to maintain.